src/hotspot/cpu/riscv/c2_MacroAssembler_riscv.cpp

```diff
@@ -1722,3 +1722,34 @@ void C2_MacroAssembler::rvv_vsetvli(BasicType bt, int length_in_bytes, Register
     }
   }
 }
+
+void C2_MacroAssembler::rvv_compare(VectorRegister vd, BasicType bt, int length_in_bytes, VectorRegister src1, VectorRegister src2, int cond, VectorMask vm) {
+  rvv_vsetvli(bt, length_in_bytes);
+  vmxor_mm(vd, vd, vd);
+  if (bt == T_FLOAT || bt == T_DOUBLE) {
+    switch (cond) {
+      case BoolTest::eq: vmfeq_vv(vd, src1, src2, vm); break;
+      case BoolTest::ne: vmfne_vv(vd, src1, src2, vm); break;
+      case BoolTest::le: vmfle_vv(vd, src1, src2, vm); break;
+      case BoolTest::ge: vmfle_vv(vd, src2, src1, vm); break;
```

Comment:
Maybe we could add some pseudo instructions like `vmfge_vv`/`vmfgt_vv` [1] .

1. https://github.com/riscv/riscv-v-spec/blob/b9afd6f5709fe3f91ce39bb83695bcfaa78eef94/v-spec.adoc?plain=1#L3676-L3681

Reply:
Fixed.